docs/Contributing/Audit-logs.md

```diff
@@ -526,9 +526,9 @@ This activity contains the following fields:
 Generated when a host is enrolled in Fleet's MDM.
 
 This activity contains the following fields:
-- "host_serial": Serial number of the host.
+- "host_serial": Serial number of the host (Apple enrollments only, always empty for Microsoft).
 - "host_display_name": Display name of the host.
-- "installed_from_dep": Whether the host was enrolled via DEP.
+- "installed_from_dep": Whether the host was enrolled via DEP (Apple enrollments only, always false for Microsoft).
```

Comment:
How does this correspond to the `installed_from_dep` field in the `host_mdm` table? 

IIRC we certain record Windows enrollments as `installed_from_dep` in that table and related API responses (or at least I believe we did at some point, but maybe that's no longer the case).

Reply:
@gillespi314 Good question, I didn't know we had that for Windows! Maybe we _could_ set this for Microsoft too in some cases? But I went with how we create that activity currently for Windows enrollments, and this field is never set: https://github.com/fleetdm/fleet/blob/6707a8b3fc3623ce0437554cb36ff2ed3cf62f48/server/service/microsoft_mdm.go#L1804-L1808